src/modules/auth/login.controller.ts

```diff
@@ -218,6 +227,15 @@ export class LoginController {
         accessTokenString
       );
 
+      if (
+        !this.tokenService.isLoginOriginMatchesRequestOrigin(
+          tokenData?.origin,
+          req
+        )
+      ) {
+        throw new Error();
```

Comment:
Actually nobody will see this error. Throwing here is just a way to return `user: null`, because this error will be caught and processed later. It is possible to achieve same without throwing, but I choose this way to be consistent with case when refresh token not verified (line 169)